lib/scrabble_word.rb

```diff
@@ -0,0 +1,48 @@
+# require "pry"
+
+class ScrabbleWord
+
+
+  def initialize (word)
+    @word = word
+    @total = 0
+  end
+
+  def word
+    return @word
+  end
+
```

Comment:
I like how you are being explicit here with your returns, just know that the last line of each method is implicitly returned in Ruby